protocol/messenger_communities.go

```diff
@@ -1005,6 +1030,7 @@ func (m *Messenger) RequestToJoinCommunity(request *requests.RequestToJoinCommun
 		CommunityID:       community.ID(),
 		SkipProtocolLayer: true,
 		MessageType:       protobuf.ApplicationMetadataMessage_COMMUNITY_REQUEST_TO_JOIN,
+		PubsubTopic:       community.PubsubTopic(), // TODO: confirm if it should be sent in community pubsub topic
```

Comment:
If we assume the owner has moved (which should be a safe assumption), it should I believe, admins should also move to, but I defer to @mprakhov @osmaczko